time-tracker-webapi/src/TimeTracker.Api/Controllers/AdminReportsController.cs

```diff
@@ -0,0 +1,31 @@
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using TimeTracker.Data;
+using TimeTracker.Library.Models;
+using TimeTracker.Library.Models.Admin;
+using TimeTracker.Library.Services;
+
+namespace TimeTracker.Api.Controllers
+{
+    [Route("admin/reports")]
+    public class AdminReportsController : Controller
+    {
+        private readonly TimeTrackerDbContext _dbContext;
+
+        public AdminReportsController(TimeTrackerDbContext dbContext)
+        {
+            _dbContext = dbContext;
+        }
+
+        /* anonymous until figure out auth with google */
+        [HttpGet(""), AllowAnonymous, ProducesResponseType(typeof(IEnumerable<UserReport>), 200)]
+        public async Task<IActionResult> AllUsersReport()
```

Comment:
I'd always coded MVC results as returning IActionResult, didn't know of different way of doing that. Will do some research to see if can return something else that will still bind with the View correctly. Only thing I'd want to avoid is creating a different class for each MVC result that is different from the ViewModel and then end up creating a lot of bloat.